metabase/metabase
# Update color range selector

Epic https://github.com/metabase/metabase/issues/22816

Updates the component used to select color ranges in table and map settings.

**How to test**
- Create a question Users, grouped by State, Sum by Count
- Visualization -> Table ->  Condition formatting -> Add rule -> Color range
- Make sure it is possible to edit the selected color range
- Visualization -> Map ->  Color
- Make sure it is possible to edit the selected color range

**Table custom formatting**
<img width="660" alt="Screenshot 2022-05-27 at 17 45 06" src="https://user-images.githubusercontent.com/8542534/170730290-ad5acc70-a5dd-4f39-be30-198d63406225.png">

**Map settings**
<img width="1728" alt="Screenshot 2022-05-30 at 15 12 21" src="https://user-images.githubusercontent.com/8542534/170990026-777dc7c6-1f06-4c26-8422-9a7e381a02a3.png">
